README.md

```diff
@@ -34,6 +34,23 @@ For pre-release Elixir versions, such as `1.11.0-rc.0`, use the full version
 specifier (`1.11.0-rc.0`). Pre-release versions are opt-in, so `1.11.x` will
 not match a pre-release.
 
+### Compatibility between Ubuntu and Erlang/OTP
+
+This list presents the known working version combos between Ubuntu
+and Erlang/OTP, as well as the non-working combos.
+
+| Ubuntu       | Erlang/OTP | Status
+|-             |-           |-
+| ubuntu-16.04 | 17 - 23    | ✅
+| ubuntu-18.04 | 17         | 🚫<sup>[1]</sup>
+| ubuntu-18.04 | 18         | 🚫<sup>[1]</sup>
+| ubuntu-18.04 | 19         | 🚫<sup>[1]</sup>
```

Comment:
I think it would be better to not list them.

Reply:
You mean have `17-19` only?

Reply:
I'm going ahead and pushing this, which I believe is the intention behind your comment.

Reply:
No, I think we should list what is compatible not what is not compatible.